native-image-tests/local-scala/src/main/scala/com/lightbend/AkkaQuickstart.scala

```diff
@@ -0,0 +1,67 @@
+package com.lightbend
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import com.lightbend.GreeterMain.SayHello
+
+object Greeter {
```

Comment:
What is the plan for these tests? Do we add something that exercise more of cluster features? Do we need 4? local-scala, local-java, cluster-scala, cluster-java? Then we probably need something in Akka Projections that covers "everything"? (the existing sample there might do that?)

Reply:
Yes, that was my thinking, to detect breakage as early and close to the broken thing as possible, we could possibly skip Java and exercise Java things from the Scala ones (extensions implemented in Java and whatever else is slightly different).

There will need to be follow up work in all Akka umbrella where we want to explicitly say native-image is supported. We may want to scope it to not cover all Akka umbrella, say we include HTTP, gRPC, R2DBC, k8 management and projections.